### PR TITLE
TE-1210 Added Slevomat rules

### PR DESCRIPTION
**Pull request by stereomon**
- Developer(s): @stereomon

- Ticket: https://spryker.atlassian.net/browse/TE-1210

- Academy PR: ACADEMY_URL_HERE

- Release Group: URL_HERE

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   CodeSniffer           | patch (currently 0.x)  |                       |

#### Release Notes

Added new rules from slevomat which add an empty line before control structures.

-----------------------------------------

### Improvement

- Added new rules from slevomat which add an empty line before control structures.

### CODE REVIEW

**Comment by dereuromark** [comment]
:-1: Too much noise. This introduces also way more newline errors than what is written above.

**Comment by stereomon** [comment]
Updated the sniffer configuration.

File: Spryker/ruleset.xml [hunk 1]
```diff
@@ -54,6 +54,20 @@
     <rule ref="SlevomatCodingStandard.Commenting.EmptyComment"/>
     <rule ref="SlevomatCodingStandard.Commenting.DisallowOneLinePropertyDocComment"/>
 
+    <rule ref="SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountBeforeFirstControlStructure"/>
+
+    <rule ref="SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountBeforeControlStructure"/>
```
Comment (dereuromark) [inline Spryker/ruleset.xml]: Why do we need these two lines?

Comment (stereomon) [reply Spryker/ruleset.xml]: I removed `IncorrectLinesCountBeforeFirstControlStructure` as we are not using it.

**Comment by stereomon** [comment]
Moved to dedicated ticket and opened a new PR for it https://github.com/spryker/code-sniffer/pull/165

**Pull request closed without merging**